model/vcs/registry.yaml

```diff
@@ -83,3 +83,24 @@ groups:
           The ID of the change (pull request/merge request) if applicable. This
           is usually a unique (within repository) identifier generated by the VCS system.
         examples: ["123"]
+      - id: vcs.change.state
```

Comment:
it seems we already have `vcs.repository.change.title` and id, which I'm suggesting in other comments to simplify to `vcs.change_request`.

If we do this, let's make sure we do it for existing attributes, otherwise, let's use `vcs.repository.change` consistently

Reply:
Since we renamed all other change related metrics and attributes to `vcs.change.*` in 536a2a7defc49a81e598b228848258ff5201e94f there's no need to change this one.